test/fixtures/es-module-loaders/register-multiple-programmatically-loaders.mjs

```diff
@@ -0,0 +1,10 @@
+import * as fixtures from '../../common/fixtures.mjs';
+import { registerLoader } from 'node:module';
+
+registerLoader(fixtures.path('es-module-loaders', 'loader-load-passthru.mjs'))
+await import(
+  `data:text/javascript,` +
+  `import { registerLoader } from %22node:module%22;` +
+  `registerLoader(%22${fixtures.path('es-module-loaders', 'loader-resolve-passthru.mjs')}%22);` +
+  `await import(%22data:text/javascript,console.log(%22Sub dynamic import%22);%22);`
```

Comment:
Also, this [test case](https://github.com/nodejs/node/pull/46826/files#diff-16536390c527c3f1d1d485d8fbc8b621a3269adbfbcb795104127194a1d56d3eR70) is not working because the quote looks like to be ignored throwing the following error:

```console
SyntaxError: Unexpected identifier 'Sub'\n    at CustomizedModuleLoader.moduleStrategy (node:internal/modules/esm/translators:116:18)\n    at CustomizedModuleLoader.moduleProvider (node:internal/modules/esm/loader:206:14)\n    at async link (node:internal/modules/esm/module_job:67:21)\n\nNode.js v21.0.0-pre\n...`
```

Reply:
I think this is not valid